Makefile

```diff
@@ -9,13 +9,20 @@ bins: temporal-server temporal-cassandra-tool temporal-sql-tool tdbg
 all: clean proto bins check test
 
 # Used in CI
-ci-build-misc: print-go-version proto go-generate buf-breaking shell-check copyright-check goimports gomodtidy ensure-no-changes
+ci-build-misc: \
+	print-go-version \
+	clean-tools \
```

Comment:
Added `clean-tools`. Why? It will do nothing in CI (those folders will be empty); but when you try to repro it locally after it failed, using `clean-tools` ensures that you're getting the latest tool versions.